assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,54 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates a module on the server.
+	 *
+	 * Activate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
+	 * @return {Promise} The dispatched action.
+	 */
 	activateModule: ( ...args ) => {
 		return Data.dispatch( STORE_NAME ).activateModule( ...args );
 	},
+	/**
+	 * Deactivates a module on the server.
+	 *
+	 * Deactivate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
+	 * @return {Promise} The dispatched action.
+	 */
 	deactivateModule: ( ...args ) => {
 		return Data.dispatch( STORE_NAME ).deactivateModule( ...args );
 	},
+	/**
+	 * Registers a module.
+	 *
+	 * @since 1.13.0
+	 * @since 1.20.0 Introduced the ability to register settings and setup components.
+	 * @since 1.22.0 Introduced the ability to add a checkRequirements function.
+	 *
+	 * @param {Object}      args                         Module action arguments.
+	 * @param {string}      args.slug                    Module slug.
+	 * @param {string}      [args.name]                  Optional. Module name. Default is the slug.
+	 * @param {string}      [args.description]           Optional. Module description. Default empty string.
+	 * @param {string}      [args.icon]                  Optional. Module icon. Default empty string.
+	 * @param {number}      [args.order]                 Optional. Numeric indicator for module order. Default 10.
+	 * @param {string}      [args.homepage]              Optional. Module homepage URL. Default empty string.
+	 * @param {WPComponent} [args.settingsEditComponent] Optional. React component to render the settings edit panel. Default none.
+	 * @param {WPComponent} [args.settingsViewComponent] Optional. React component to render the settings view panel. Default none.
+	 * @param {WPComponent} [args.setupComponent]        Optional. React component to render the setup panel. Default none.
+	 * @param {Function}    [args.checkRequirements]     Optional. Function to check requirements for the module. Throws a WP error object for error or returns on success.
+	 * @return {Promise} The dispatched action.
+	 */
 	registerModule: ( ...args ) => {
```

Comment:
See above, let's change this to `( slug, settings )` and copy over the parameter docs from the action.